webpack/webpack.config.base.js

```diff
@@ -86,6 +86,8 @@ const aliases = {
   '@stacks/transactions': '@stacks/transactions/dist/esm',
   '@stacks/wallet-sdk': '@stacks/wallet-sdk/dist/esm',
   'leather-styles': path.resolve('leather-styles'),
+  'react': path.resolve('./node_modules/react'),
```

Comment:
We need to alias `react` + `react-dom` to fix problems with different versions of react and hooks errors. 

![Screenshot 2024-06-04 at 09 51 49](https://github.com/leather-wallet/extension/assets/2938440/eb401397-5742-46e7-8d7c-c7d3f22eeb3c)
![Screenshot 2024-06-04 at 12 09 43](https://github.com/leather-wallet/extension/assets/2938440/1159970b-bf8e-4985-8b49-a141cab8e699)


See [this article](https://blog.maximeheckel.com/posts/duplicate-dependencies-npm-link/) for a good explanation. 

Thanks to Kyran for suggesting this aliasing.